lib/bundles/inspec-compliance/cli.rb

```diff
@@ -30,26 +30,22 @@ def login(server) # rubocop:disable Metrics/AbcSize
       url = options['server'] + options['apipath']
       if !options['user'].nil? && !options['password'].nil?
         # username / password
-        success, msg = login_legacy(url, options['user'], options['password'], options['insecure'])
+        msg = login_legacy(url, options['user'], options['password'], options['insecure'])
       elsif !options['user'].nil? && !options['token'].nil?
         # access token
-        success, msg = store_access_token(url, options['user'], options['token'], options['insecure'])
+        msg = store_access_token(url, options['user'], options['token'], options['insecure'])
       elsif !options['refresh_token'].nil? && !options['user'].nil?
         # refresh token
-        success, msg = store_refresh_token(url, options['refresh_token'], true, options['user'], options['insecure'])
+        msg = store_refresh_token(url, options['refresh_token'], true, options['user'], options['insecure'])
         # TODO: we should login with the refreshtoken here
       elsif !options['refresh_token'].nil?
-        success, msg = login_refreshtoken(url, options)
+        msg = login_refreshtoken(url, options)
       else
         puts 'Please run `inspec compliance login` with options --token or --refresh_token and --user'
         exit 1
       end
 
-      if success
-        puts '', 'Successfully authenticated'
-      else
-        puts msg
-      end
+      puts '', msg
```

Comment:
So, I wasn't sure about doing the extra blank line or not, it's just a user-readability thing.  When you paste that big token in and then it spits out the message, it's kinda hard (imo) to see the message under the block of text, hence the extra line, but I'm not attached to it, so if ppl think it's fine the way it is/if it's not a generally cool thing to spit out an extra line for something like this, I would be happy to remove the extra line.